src/gov/nist/core/LexerCore.java

```diff
@@ -238,7 +245,12 @@ public Token match(int tok) throws ParseException {
                 this.currentMatch = new Token();
                 this.currentMatch.tokenValue = id;
                 this.currentMatch.tokenType = ID_NO_WHITESPACE;
-            } else {
+            } else if (tok == IPV6){
```

Comment:
if you change the IPV6 token code above END; you may need to move this case to "characted class" a bit below..